.gitignore

```diff
@@ -25,3 +25,4 @@ __snapshots__
 
 # debug & other vs code settings
 .vscode
+*.env
```

Comment:
We don't want to ignore the sample env, I would revert this change please.

Reply:
noted with thanks